### PR TITLE
makes the "sexy" variants of the mime/clown suit the "skirt" variant of them

### PR DESCRIPTION
**Pull request by ToasterBan**
## About The Pull Request

as the title says, makes the "sexy" clown / mime outfit into "skits"

## Why It's Good For The Game

tbh the sexy mime outfit looks better than the mime jumpskirts also I thought it would be funny if the sexy clown outfit could be roundstart

## Changelog
:cl: Bluezorua
del: Removed mime jumpskirt
tweak: "Sexy" variations of the mime/clown suit can be spawned roundstart trough the jumpsuit/jumpskirt char setup option
/:cl:

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
but it still says sexy on the examine

File: code/modules/clothing/under/jobs/civilian/clown_mime.dm [hunk 1]
```diff
@@ -6,17 +6,17 @@
...
-/obj/item/clothing/under/rank/civilian/mime/skirt
+/*/obj/item/clothing/under/rank/civilian/mime/skirt
 	name = "mime's skirt"
 	icon_state = "mime_skirt"
 	item_state = "mime"
 	item_color = "mime_skirt"
 	body_parts_covered = CHEST|GROIN|ARMS
 	can_adjust = FALSE
 	fitted = FEMALE_UNIFORM_TOP
+*/
```
Comment (Archanial) [inline code/modules/clothing/under/jobs/civilian/clown_mime.dm:17]: if it's not needed remove it

**Comment by ivanmixo** [comment]
The sexy mime skirt looks absolutely awful, so this is kinda insulting to be honest

image

**Comment by PowerfulBacon** [comment]
> 
> 
> The sexy mime skirt looks absolutely awful, so this is kinda insulting to be honest
> 
> image

All the mime clothes look absolutely awful

**Comment by ghost** [comment]
Bad PR. Reduces RP. I want to RP as the mime dash sisters god dammit

**Comment by ItsTisme** [comment]
> Bad PR. Reduces RP. I want to RP as the mime dash sisters god dammit

I hate that I know where this is from, and may be very dubius to do

**Comment by ToasterBan** [comment]
> Bad PR. Reduces RP. I want to RP as the mime dash sisters god dammit

stop being horny for the fr*nch

**Comment by ToasterBan** [comment]
this was a shit discord joke gone wrong

**Pull request closed without merging**